spec/models/user_spec.rb

```diff
@@ -1,11 +1,13 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  before { @user = User.new(
+  before do
+    @user = User.new(
         name: '星宮いちご',
         email: 'i-hoshimiya@star-light.ac',
         password: 'foobar',
-        password_confirmation: "foobar") }
+        password_confirmation: 'foobar')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.